### ISSUE TITLE
Missing NOT operator

### ISSUE THREAD

**Issue by adm244** [alekseynemiro/nemiro.oauth.dll#37]
I think this line meant to be "`if (!AccessToken.IsNullOrEmpty(value))`":
https://github.com/alekseynemiro/nemiro.oauth.dll/blob/a0e4b21dd361107636c05f44baf45391d06e0178/src/Nemiro.OAuth/OAuthBase.cs#L400

**Comment by alekseynemiro** [comment]
No. If the incoming value is empty, then the value from the current instance of the class will be used.
